### PR TITLE
Fix clippy warnings

### PR DESCRIPTION
**Pull request by na-g**
This fixes the `cargo clippy` warnings with version `clippy 0.1.69 (11d96b5 2023-02-01)`.

### CODE REVIEW

**Comment by na-g** [comment]
We'll need to bump the minimum rust version to land this.

**Review by mfromanmoz — APPROVED** [review]

File: examples/file_parser.rs [hunk 1]
```diff
@@ -61,7 +61,7 @@ fn main() {
     let res = webrtc_sdp::parse_sdp(&s, true);
     match res {
         Err(why) => panic!("Failed to parse SDP with error: {}", why),
```
Comment (nils-ohlmeier) [inline examples/file_parser.rs:63]: Clippy isn't complaining about the `why` not being inside the panic message?

**Review by nils-ohlmeier — APPROVED** [review]
LGTM

File: examples/file_parser.rs (hunk 1 above)
Comment (na-g) [reply examples/file_parser.rs:63]: That is a good question ...

Comment (na-g) [reply examples/file_parser.rs:63]: @nils-ohlmeier apparently `panic!` and `format!` have different string interpolation concerns:
```
warning: panic message contains unused formatting placeholders
  --> examples/file_parser.rs:29:44
   |
29 |         Err(why) => panic!("Failed to open {display}: {why}"),
   |                                            ^^^^^^^^^  ^^^^^
   |
   = note: this message is not used as a format string when given without arguments, but will be in Rust 2021
   = note: `#[warn(non_fmt_panics)]` on by default
help: add the missing arguments
   |
29 |         Err(why) => panic!("Failed to open {display}: {why}", ...),
   |                                                             +++++
```

Comment (nils-ohlmeier) [reply examples/file_parser.rs:63]: Then we will wait till Rust 2021 I guess :-)

**Pull request merged**